docs/reference/robot.md

```diff
@@ -43,7 +43,7 @@ The corresponding command line instruction will have the form:
 
     ``<controller_program_name> "controllerArgs[0]" "controllerArgs[1]" ...``
 
-    In case of C, C++, and Java controller programs the values are passed as arguments of the `main` function or method.
+    In case of C, C++, python and Java controller programs the values are passed as arguments of the `main` function or method.
```

Comment:
We usually spell Python with an uppercase letter in the documentation.
```suggestion
    In case of C, C++, Python and Java controller programs the values are passed as arguments of the `main` function or method.
```